test/functional/api/mark_extended_spec.lua

```diff
@@ -507,7 +507,7 @@ describe('API/extmarks', function()
     feed('0l3dl<esc>')
     check_undo_redo(ns, marks[1], 0, 2, 0, 1)
     check_undo_redo(ns, marks[2], 0, 3, 0, 1)
-    -- delete 1, nothing should happend to our marks
+    -- delete 1, nothing should happened to our marks
```

Comment:
happen reads better here than happened.
```suggestion
    -- delete 1, nothing should happen to our marks
```

Reply:
I will change to your suggestion.